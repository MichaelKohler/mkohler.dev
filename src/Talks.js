import React from 'react';

const Talks = () => {
  return (
    <p>Talks..</p>
  );
};

export default Talks;
