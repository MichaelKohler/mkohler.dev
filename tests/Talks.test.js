import React from 'react';
import { shallow } from 'enzyme';

import Talks from '../src/Talks';

test('should correctly render Talks', () => {
  const component = shallow(<Talks />);
  expect(component).toMatchSnapshot();
});
