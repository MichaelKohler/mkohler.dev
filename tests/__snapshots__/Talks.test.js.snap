// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should correctly render Talks 1`] = `
<section
  id="talks"
>
  <h2>
    Talks
  </h2>
  <h3
    className="quote"
  >
    I give talks about all different kinds of topics. You can find all slides here.
  </h3>
  <h3>
    Mozilla General
  </h3>
  <ul
    className="talks-list"
  >
    <li>
      <a
        href="/presentations/mozillaCH-community-meetup-2017/index.html"
        target="_blank"
      >
        Mozilla Switzerland Community Meetup, Zurich, 28.01.2017
      </a>
    </li>
    <li>
      <a
        href="/presentations/mozillaCH_intro_mozilla/index.html"
        target="_blank"
      >
        Mozilla Intro & Update, Mozilla Switzerland at TechUP, Luzern, 13.04.2016
      </a>
    </li>
    <li>
      <a
        href="/presentations/mozillaCH-day-whats-new-in-mozilla/index.html"
        target="_blank"
      >
        Mozilla Day Switzerland 2015 - What's new in Mozilla?, Mozilla Switzerland, Zurich, 17.10.2015
      </a>
    </li>
    <li>
      <a
        href="/presentations/mozweekend-berlin-2015-contributing/index.html"
        target="_blank"
      >
        Mozilla Weekend Berlin 2015 - Contributing at Mozilla, German-speaking Mozilla Community, Berlin, 11.07.2015
      </a>
    </li>
    <li>
      <a
        href="/presentations/mozweekend-berlin-2015-outro/index.html"
        target="_blank"
      >
        Mozilla Weekend Berlin 2015 - Outro Day 1, German-speaking Mozilla Community, Berlin, Germany, 11.07.2015
      </a>
    </li>
    <li>
      <a
        href="/presentations/mozweekend-berlin-2015-intro/index.html"
        target="_blank"
      >
        Mozilla Weekend Berlin 2015 - Intro, German-speaking Mozilla Community, Berlin, Germany, 11.07.2015
      </a>
    </li>
    <li>
      <a
        href="/presentations/mozillaCH-how-mozilla-protect-web/index.html"
        target="_blank"
      >
        How does Mozilla protect the open web?, Mozilla Switzerland, Bern, Switzerland, 08.06.2015
      </a>
    </li>
    <li>
      <a
        href="/presentations/mozilla-switzerland/index.html"
        target="_blank"
      >
        Some information about Mozilla Switzerland, Mozilla Berlin, 20.02.2015
      </a>
    </li>
    <li>
      <a
        href="/presentations/hacking-day-berlin-okt-2014/index.html"
        target="_blank"
      >
        Introduction - Mozilla Hacking Day Berlin, Berlin, 11.10.2014
      </a>
    </li>
    <li>
      <a
        href="/presentations/hacking-day-berlin-okt-2014-closing/index.html"
        target="_blank"
      >
        Closing - Mozilla Hacking Day Berlin, Berlin, 11.10.2014
      </a>
    </li>
  </ul>
  <h3>
    Firefox
  </h3>
  <ul
    className="talks-list"
  >
    <li>
      <a
        href="https://docs.google.com/presentation/d/1TvIpFfZ-2qAcWOgxCGmJjjo55SNxGeIoFJqA4AV2uyM"
        target="_blank"
      >
        Firefox - Quantum Release Party, Mozilla Switzerland, 15.11.2017
      </a>
    </li>
    <li>
      <a
        href="https://docs.google.com/presentation/d/1J2_eCtEyE1Ss3U0GETHwtQ0Jw63b2tb0QByACwVQRDI/"
        target="_blank"
      >
        Firefox - Quantum/Photon, Mozilla Switzerland, 18.05.2017
      </a>
    </li>
    <li>
      <a
        href="https://docs.google.com/presentation/d/11euVpPoJ7kKO1OcWTYCqA3XOmUC_DAoGi1rJEeQtOS8/"
        target="_blank"
      >
        Firefox Hackathon, Mozilla Switzerland, 30.04.2016
      </a>
    </li>
    <li>
      <a
        href="/presentations/mozillaCH-day-devtools-demo/index.html"
        target="_blank"
      >
        Mozilla Day Switzerland 2015 - Firefox Dev Tools Demo, Mozilla Switzerland, Zurich, 17.10.2015
      </a>
    </li>
    <li>
      <a
        href="/presentations/mozweekend-berlin-2015-new-in-firefox/index.html"
        target="_blank"
      >
        Mozilla Weekend Berlin 2015 - What's new in Firefox?, German-speaking Mozilla Community, Berlin, 11.07.2015
      </a>
    </li>
    <li>
      <a
        href="/presentations/mozillaCH-why-should-you-use-firefox/index.html"
        target="_blank"
      >
        Why should you use Firefox?, Mozilla Switzerland, Zürich, Switzerland, 03.06.2015
      </a>
    </li>
    <li>
      <a
        href="/presentations/fx10-zurich/index.html"
        target="_blank"
      >
        10 years of Firefox, Zurich, 17.11.2014
      </a>
    </li>
    <li>
      <a
        href="/presentations/mozilla_weekend_of_code_2012.html"
        target="_blank"
      >
        Firefox Source Code 101, Mozilla Weekend of Code 2012, Schlieren ZH
      </a>
    </li>
  </ul>
  <h3>
    Connected Devices & Firefox OS
  </h3>
  <ul
    className="talks-list"
  >
    <li>
      <a
        href="https://docs.google.com/presentation/d/1J6vTfZYunNT9nbvzmGkz0kq_p5m2HL8DDOb1Dg6wO3M"
        target="_blank"
      >
        Connected Devices and Smart Home, Mozilla Switzerland at techmeetings.ch, Fribourg, 31.03.2016
      </a>
    </li>
    <li>
      <a
        href="https://docs.google.com/presentation/d/1y9E_KJ0v5P3LYoklqC1Gbt-drGXRJyu2bX9J6kD1lDY"
        target="_blank"
      >
        Firefox OS and Connected Devices, Mozilla Switzerland, Zürich, 09.02.2016
      </a>
    </li>
    <li>
      <a
        href="https://docs.google.com/presentation/d/1T8PbAqkCepClJz3OkTpKkrs01TVCTYW1K1hzT0ImC6s/"
        target="_blank"
      >
        Firefox OS and Connected Devices, Mozilla Switzerland, Fribourg, 01.02.2016 (not delivered :()
      </a>
    </li>
    <li>
      <a
        href="/presentations/mozillaCH-mozilla-intro-firefox-os/index.html"
        target="_blank"
      >
        WebMardi Lausanne 2015 - Intro to Mozilla / Firefox OS, Mozilla Switzerland, Lausanne, 10.11.2015
      </a>
    </li>
    <li>
      <a
        href="/presentations/oscal-firefoxos-hacking/index.html"
        target="_blank"
      >
        Creating Firefox OS Apps and hacking Firefox OS itself!, OSCAL 2015, Tirana, Albania, 09.05.2015
      </a>
    </li>
    <li>
      <a
        href="/presentations/mozilla-mobilecamp-dresden-2015-hacking/index.html"
        target="_blank"
      >
        Firefox OS - Creating your own app and hack Firefox OS itself, MobileCamp Dresden 2015, Dresden, 25.04.2015
      </a>
    </li>
    <li>
      <a
        href="/presentations/mozilla-mobilecamp-dresden-2015/index.html"
        target="_blank"
      >
        Firefox OS - still advancing, MobileCamp Dresden 2015, Dresden, 25.04.2015
      </a>
    </li>
    <li>
      <a
        href="/presentations/mozillach-combining-firefox-os-webmaker/index.html"
        target="_blank"
      >
        Combining Firefox OS and Webmaker, MozillaCH, Zürich, 26.03.2015
      </a>
    </li>
    <li>
      <a
        href="http://slides.com/michaelkohler/hacking-on-gaia#/"
      >
        Hacking on Gaia, FOSS Asia, 15.03.2015
      </a>
    </li>
    <li>
      <a
        href="/presentations/fossasia-combining-firefox-os-webmaker/index.html"
        target="_blank"
      >
        Combining Firefox OS and Webmaker, FOSS Asia, 14.03.2015
      </a>
    </li>
    <li>
      <a
        href="/presentations/firefoxosappday-frankfurt/index.html"
        target="_blank"
      >
        Firefox OS App Day Frankfurt, Frankfurt, 22.11.2014
      </a>
    </li>
    <li>
      <a
        href="/presentations/ffos-introduction-to-firefoxos-dev/index.html"
        target="_blank"
      >
        TechUP July 2014 - Introduction to Firefox OS Development, TechUP Luzern, 02.07.2014
      </a>
    </li>
    <li>
      <a
        href="/presentations/new-in-firefoxos-appday2014/index.html"
        target="_blank"
      >
        App Day 2014 - New in Firefox OS, Mozilla Firefox OS App Day Zurich, 14.06.2014
      </a>
    </li>
    <li>
      <a
        href="/presentations/ffos-appday2014-switzerland-mozch/index.html"
        target="_blank"
      >
        Mozilla CH / App Day Intro, Mozilla Firefox OS App Day Zurich, 14.06.2014
      </a>
    </li>
    <li>
      <a
        href="/presentations/mobilecamp-dresden-2014/index.html"
        target="_blank"
      >
        Firefox OS - Introduction, MobileCamp Dresden 2014, 10.05.2014
      </a>
    </li>
    <li>
      <a
        href="/presentations/ffos-appday-duesseldorf/index.html"
        target="_blank"
      >
        History of Firefox OS, Firefox OS App Day Düsseldorf, net-m, 15.11.2013
      </a>
    </li>
    <li>
      <a
        href="/presentations/ffos-appday-duesseldorf-coding/index.html"
        target="_blank"
      >
        Creating a Firefox OS App - Live coding, Firefox OS App Day Düsseldorf, net-m, 15.11.2013
      </a>
    </li>
    <li>
      <a
        href="/presentations/ffos-appday-switzerland/FFOSAppDay.html"
        target="_blank"
      >
        Firefox OS App Day Zurich - Event, Schlieren ZH, 25.05.2013
      </a>
    </li>
    <li>
      <a
        href="/presentations/ffos-appday-switzerland-mozch/MozillaCH.html"
        target="_blank"
      >
        Firefox OS App Day Zurich - MozillaCH, Schlieren ZH, 25.05.2013
      </a>
    </li>
  </ul>
  <h3>
    Community Building
  </h3>
  <ul
    className="talks-list"
  >
    <li>
      <a
        href="/presentations/oscal-community-building/index.html"
        target="_blank"
      >
        Communities are easy to build but hard to maintain, OSCAL 2015, Tirana, Albania, 10.05.2015
      </a>
    </li>
  </ul>
  <h3>
    Other
  </h3>
  <ul
    className="talks-list"
  >
    <li>
      <a
        href="https://docs.google.com/presentation/d/1yPy7oLr1u9RyyaF-CS7Yf3gDX873bJI82LgLIZPisnQ/"
        target="_blank"
      >
        FFHS Raspberry Pi Projektpräsentation (German), 14.04.2016
      </a>
    </li>
    <li>
      <a
        href="/presentations/cern-webfest/Webfest.html"
        target="_blank"
      >
        CERN Webfest - Mozilla Webmaker, CERN, 02.08.2013
      </a>
    </li>
  </ul>
</section>
`;
